admin-ui/src/pages/settings-page.tsx

```
@@ -806,16 +988,156 @@ function ExtraPromptsCard({
+              })
+            )}
+
+            <Button type="button" variant="outline" size="sm" onClick={onAddItem}>
+              Add alias
+            </Button>
+          </div>
+        )}
+      </CardContent>
+    </Card>
+  )
+}
```

ESLint

[error] Function 'ModelAliasesCard' has too many lines (117). Maximum allowed is 100.

(max-lines-per-function)